### PR TITLE
62668 - Update the URL and title page for the COVID-19 vaccine appointment workflow

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Summary
This PR updates the URL and title page for the COVID-19 vaccine appointment workflow.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#62668

## Testing done
Unit and e2e testing

## Screenshots
**Scenario 1**
Screenshot 2023-09-21 at 12 46 26 PM
Screenshot 2023-09-21 at 12 54 49 PM
Screenshot 2023-09-21 at 12 45 58 PM

**Scenario 2**
Screenshot 2023-09-21 at 12 46 26 PM
Screenshot 2023-09-21 at 12 46 19 PM
Screenshot 2023-09-21 at 12 46 42 PM
Screenshot 2023-09-21 at 12 46 51 PM
Screenshot 2023-09-21 at 12 47 04 PM
Screenshot 2023-09-21 at 12 47 10 PM
Screenshot 2023-09-21 at 12 47 22 PM
Screenshot 2023-09-21 at 12 47 28 PM
Screenshot 2023-09-21 at 12 47 50 PM

## What areas of the site does it impact?
VAOS

## Acceptance criteria
Background: the va_online_scheduling_breadcrumb_url_update toggle is on

Given the user is on Upcoming VA appointments homepage,
And the user clicks on Start Scheduling button for COVID-19 vaccine appointment then lands on

- [ ] COVID-19 vaccine appointment plan ahead page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine
- [ ] Have you received a COVID-19 vaccine? page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/doses-received
- [ ] When user selects Yes and lands on the We can’t schedule your second dose online page, then URL will be https://va.gov/my-health/appointments/schedule/covid-vaccine/contact-facility
- [ ] When user selects No/I'm not sure, then lands on the Choose a location page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/location
- [ ] Choose a clinic, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/clinic
- [ ] Choose a date page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/date-time
- [ ] When to plan for a second dose page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/second-dose
- [ ] Confirm your contact information then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/contact-information
- [ ] Review your appointment details page, then URL will display https://va.gov/my-health/appointments/schedule/covid-vaccine/review
- [ ] Once the user schedules a COVID-19 vaccine appointment, then URL for the confirmation state will display [https://va.gov/my-health/appointments/[ID]?confirmMsg=true](https://va.gov/my-health/appointments/%5BID%5D?confirmMsg=true)

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by jenniemc — COMMENTED** [review]
The `edit` contact link in the COVID review page does not work. Aside from that everything else looks good.

File: src/applications/vaos/covid-19-vaccine/flow.js [hunk 1]
```diff
@@ -81,3 +82,85 @@ export default {
...
+        ? 'contact-facility'
+        : '/new-covid-19-vaccine-appointment/contact-facility',
+    },
+    contactInfo: {
+      ...flow.contactInfo,
+      url: featureBreadcrumbUrlUpdate
+        ? 'contact-information'
+        : '/new-covid-19-vaccine-appointment/contact-info',
+    },
+    home: {
```
Comment (jenniemc) [inline src/applications/vaos/covid-19-vaccine/flow.js:118]: Using the attribute `home` and `root` have a similar connotation. How about changing `home` to `typeOfCare` or  `chooseCare` or `scheduleCare`

**Review by jenniemc — APPROVED** [review]
Looks good to me! I have one suggestion.

File: src/applications/vaos/covid-19-vaccine/flow.js (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/covid-19-vaccine/flow.js:118]: @jenniemc - Great suggestion!

**Review by simiadebowale — APPROVED** [review]
LGTM!

**Pull request merged**